docs/aws-setup-guide.md

```
@@ -0,0 +1,574 @@
+
+##### 트러블슈팅
+
+| 증상 | 원인 | 해결 |
+|------|------|------|
+| `Connection timed out` | 보안 그룹 규칙 미설정 | RDS 보안 그룹에 EC2 보안 그룹 ID 추가 |
+| `Connection refused` | RDS 실행 안 됨 | RDS 인스턴스 상태 확인 |
+| `password authentication failed` | 잘못된 비밀번호 | RDS 마스터 비밀번호 확인 |
+| `could not translate host name` | 잘못된 엔드포인트 | RDS 엔드포인트 주소 재확인 |
+
+#### 2.3. 연결 정보
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)